### DISCUSSION TITLE
Unable to synch Thunderbird Calendar

### DISCUSSION THREAD

**Discussion by philips791** [kewisch/gdata-provider#515] [Support] [answered]
After finally giving up the continuing problem with the Google still has not changing the rate limit and the Google calendar update process taking over my PC, exceeding the maximum  RAM available, and interrupting even simple things like composing an email message.  I deleted my Google calendar and am only using my Home calendar.  I exported from the Google calendar the ics file and imported it into the Home calendar within Thunderbird.  However now I am unable to Synchronize the Home calendar with my Google calendar using the add-on Provider for Google Calendar.  The Synchronize icon is now greyed out even though it is installed.  It has been months since I installed the add-on so perhaps I have disabled in removing the Google calendar from TB?  I appreciate your assistance.  Thank you.

**Comment by xjonx1** [comment]
The app has to be updated. The developer has been contacted. The problem is the app needs google's new OAuth 2.0.

**Reply by Frenchwolfie** [reply]
The problem is still the same for me. W10 64b 22H2 build 19045.2193, Thunderbird 102.4.2 64b.

**Comment by xjonx1** [comment, marked as answer]
There was an issue with the add-on and is now fixed by the developer. Details here: https://github.com/kewisch/gdata-provider/discussions/519#discussioncomment-4147395
